### ISSUE TITLE
ST PASA Datetime validation bug

### ISSUE THREAD

**Issue by prakaa** [UNSW-CEEM/NEMSEER#24]
Forecasted end was previously assessed based on increment from forecasted_start.

Instead, it should be validated based on its on "latest forecast time"

**Comment by prakaa** [comment]
Fixed in 406d80cdeba19bde885f365becd9e78054fa85e6, will close once PR is merged into master

**Comment by prakaa** [comment]
Closed with #28
